### PR TITLE
MAT-8017 - move intendedVenue into MeasureMetaData class

### PR DESCRIPTION
**Pull request by etan-sb**
## MADiE PR Template

Jira Ticket: [MAT-8017](https://jira.cms.gov/browse/MAT-8017)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e. Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/models/measure/Measure.java [hunk 1]
```diff
@@ -49,6 +50,7 @@
   @Type(value = QdmMeasure.class, name = "QDM v5.6")
 })
 @ValidLibraryName
+@ValidIntendedVenue
```
Comment (sb-prateekkeerthi) [inline src/main/java/gov/cms/madie/models/measure/Measure.java:53]: shouldn't we add this in MeasureMetaData.java file not at the measure file

**Review by sb-prateekkeerthi — CHANGES_REQUESTED** [review]

File: src/main/java/gov/cms/madie/models/measure/Measure.java (hunk 1 above)
Comment (etan-sb) [reply src/main/java/gov/cms/madie/models/measure/Measure.java:53]: Resolved with @sb-prateekkeerthi. This should be in the Measure level

**Review by adongare — APPROVED** [review]

**Review by sb-prateekkeerthi — APPROVED** [review]
